### ISSUE TITLE
adding shrinkwarp to a postshinkwarp hook results in sh: 1: not found

### ISSUE THREAD

**Issue by graingert** [evocateur/shrinkwarp#1]
`sh: 1: shrinkwarp: not found`

Running node 6.9 LTS

**Comment by graingert** [comment]
https://github.com/graingert/shrinkwrap-fail

**Comment by evocateur** [comment]
@graingert Thanks for the brilliant repro case!

However, this is actually an issue with npm v3.10.8, where shrinkwrap lifecycles aren't being executed from the proper directory. I indirectly helped fix it in https://github.com/npm/npm/pull/13942, which was published in npm v3.10.9.

Unfortunately, I have no idea when node v6 LTS will pick up npm v3.10.9. If you're unable to upgrade npm (`npm -g i npm@3`), you can workaround it in your scripts with some moderate hackery (as outlined in [this comment](https://github.com/npm/npm/pull/13942#issuecomment-246744626)):

``` json
"scripts": {
  "postshrinkwrap": "cd $(npm prefix) && $(npm bin)/shrinkwarp --ignore fsevents"
}
```

**Comment by graingert** [comment]
meh, I think I'll just use yarn. Thanks though!
